ReportingServicesTools/Functions/CatalogItems/Rest/Write-RsRestCatalogItem.ps1

```diff
@@ -110,8 +110,12 @@ function Write-RsRestCatalogItem
         else
         {
             $catalogItemsByPathApi = $ReportPortalUri + "api/$RestApiVersion/CatalogItems(Path='{0}')"
+            $powerBIReportsByPathApi = $ReportPortalUri + "api/$RestApiVersion/PowerBIReports(Path='{0}')/Model.Upload"
         }
         $catalogItemsUpdateUri = $ReportPortalUri + "api/$RestApiVersion/CatalogItems({0})"
+
+        $maxFileSizeInMb = 2000
```

Comment:
i recommend to make this parameters with default values so you can test it with smaller files